### PR TITLE
Making changes on NewBranch

### PR DESCRIPTION
**Pull request by GoldenCave**
Made changes to README again! For educational purposes.

### CODE REVIEW

**Comment by erikvanmechelen** [comment]
what's up

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # hello_world
 Demonstrating Basics of Git
+
+Making changes
```
Comment (GoldenCave) [inline README.md]: Writing a comment here

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # hello_world
 Demonstrating Basics of Git
```
Comment (erikvanmechelen) [inline README.md:2]: what's up

Comment (GoldenCave) [reply README.md:2]: Hello reviewer.

Comment (GoldenCave) [reply README.md:2]: This is resolved.

**Comment by GoldenCave** [comment]
Merging with the NewBranch.

**Pull request merged**